### DISCUSSION TITLE
lower_camel_case

### DISCUSSION THREAD

**Discussion by slhoehn** [cargo-generate/cargo-generate#806] [Q&A] [answered]
Some of the string methods such as lower_camel_case are not functioning

https://github.com/cargo-generate/cargo-generate/pull/662/files#diff-64c8ff2888d7415ece0c26f674068d7dbb148196c7c775c15680e98e05a91711

**Comment by sassman** [comment]
those functions are for the rhai script bindings. You could open an issue and prove more context. Then we might find the problem.

**Comment by taurr** [comment, marked as answer]
@sassman is correct - those methods are for `rhai` scripting - you might be looking for the [filters](https://shopify.dev/api/liquid/filters/camelize) in the `liquid` template.

**Reply by sassman** [reply]
But the question reveals a certain issue: the functions available in rhai and the filters we are providing are not fully in sync.

So it's maybe worth to have an issue that deals with feature parity for rhai and liquid (functions vs filters probably)

**Reply by taurr** [reply]
Not so sure we should try to align it!
`Liquid` is moving on its own pace here (as far as I remember, some of the case functions weren't even available when I implemented the `rhai` extensions - some still aren't).

We should - nai, must - do something about the documentation though. It should be abundantly clear that the 2 languages are used for 2 different purposes and uses 2 different syntax's, hence they will not have the same capabilities!
